extensions/2.0/Vendor/FB_materials_lightmap/README.md

```diff
@@ -0,0 +1,76 @@
+# FB\_materials\_lightmap
+
+## Contributors
+
+* Alfred Young, Facebook, [@idorurez](https://github.com/idorurez)
+* Pär Winzell, Facebook, [@zellski](https://twitter.com/zellski)
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension adds an additional annotation to a material, providing a means to introduce a lightmap to be applied in the same scope.
+
+Some proven real-world applications for this information:
+1. Having multiple variations of lighting scenarios without having to bake lighting into a single, original base color/texture.
+2. Specify a separate UV channel (texCoord), to vary the new texture texel resolution with respect to the texture it's being applied to.
+3. Modulate the intensity of a lightmap, dimming or increasing it's effect.
+
+## Specifying Material extension
+
+The data is defined as an optional extension to a glTF `materials`, by adding an `extensions` property and defining a `FB_materials_lightmap` property with a `lightmapTexture` node inside it:
```

Comment:
I think we can confidently say that the `lightmapTexture` must be sRGB, right?

Reply:
Or maybe not? See Unity docs and HDR / BC6H references..

Reply:
> My thought is that consider whatever platform this gets ingested on, it would be advantageous to leave color space to the adopter's discretion.

We have tried to be specific about the colorspace of other textures in the glTF spec; unless there's a clear reason not to I think we should do the same here. The colorspace could depend on the texture format if necessary.

Reply:
A lightmap technically can be unbounded in range (thus HDR) because a light map represents a measure of the incoming light.